docs/source/support/frequently-asked-questions.rst

```diff
@@ -578,6 +578,44 @@ Here we provide a bit more context on how the ``run:`` commands are used. The ex
 
 After this step, the reusable workflow then runs the ``pytest`` command. To see the full reusable workflow file, please visit https://github.com/scikit-package/release-scripts/blob/main/.github/workflows/_tests-on-pr.yml.
 
+.. _faq-private-repo-no-codecov:
+
+I have a private repo and don't have a Codecov paid plan.  Can I modify the CI workflows for this situation?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+By default, the workflows in ``tests-on-pr.yml`` and ``matrix-and-codecov-on-merge-to-main.yml`` will have a section that uploads the code coverage to Codecov, and CI will fail if unsuccessful. If you have a private repository and Codecov cannot be run, CI will always fail, and this is undesirable.
+
+In addition to being able to customize additional commands to be ran in the ``run:`` section (see :ref:`faq-github-actions-extra-cli-commands` above), you can also customize the workflows that you want your repository to run. On top of the ``tests-on-pr.yml`` and ``matrix-and-codecov-on-merge-to-main.yml`` workflows in the ``scikit-package/release-scripts`` repository, we offer ``tests-on-pr-no-codecov.yml`` and ``matrix-no-codecov-on-merge-to-main.yml`` for users in this situation.
```

Comment:
ran -> run
customize -> change
"On top of..."  -> "By default, scikit-package will run the ``tests-on-pr.yml`` and ``matrix-and-codecov-on-merge-to-main.yml`` workflows in the ``scikit-package/release-scripts`` repository.  It is possible to override this default and run a different workflow for your CI if it is available in ``scikit-package/release-scripts``.  For example, for the situation above we offer we offer ``tests-on-pr-no-codecov.yml`` and ``matrix-no-codecov-on-merge-to-main.yml`` which run the tests without Codecov.